common/src/sql/dbinit.sql

```diff
@@ -772,6 +774,9 @@ CREATE TABLE omicron.public.snapshot (
     /* Every Snapshot consists of a root volume */
     volume_id UUID NOT NULL,
 
+    /* Where will the scrubbed blocks eventually land? */
+    destination_volume_id UUID,
```

Comment:
Why is this optional?

Reply:
This will be populated while the scrub is occurring (from volume_id to destination_volume_id), and None when the scrub has completed (in that case, volume_id will be set to destination_volume_id, and the volume at volume_id will be deleted.